src/Sylius/Behat/Page/Admin/ProductVariant/UpdatePage.php

```diff
@@ -75,6 +75,17 @@ public function specifyCurrentStock(int $amount): void
         $this->getElement('on_hand')->setValue($amount);
     }
 
+    public function selectOption(string $optionName, string $optionValue): void
+    {
+        $optionName = strtoupper($optionName);
```

Comment:
According to [this comment](https://github.com/Sylius/Sylius/pull/10364#discussion_r282037449) - this is a new function you use only [there](https://github.com/Sylius/Sylius/pull/10364/files#diff-9caa6c41a42784d0658dcbb0dbc447eaR107), as far as I see? So `strtoupper` should be called in a context function, not in a page function